face_api_dataset/dataset.py

```diff
@@ -58,21 +62,53 @@ class Modality(Enum):
     
     **Type**: `ndarray[uint16]`. **Channels**: 1.
     """
-    LANDMARKS = auto()
+    LANDMARKS_IBUG68 = auto()
+    """
+    iBUG-68 landmarks. Each landmark is given by name and two coordinates (name, x,y) in pixels.
+    Each keypoint is a 2D projection of a 3D landmark. 
+    
+    **Type**: `List[Type[str, int, int]`.
+    """
+    LANDMARKS_CONTOUR_IBUG68 = auto()
     """
-    iBUG-68 landmarks. Each landmark is given by two coordinates (x,y) in pixels.
+    iBUG-68 contour landmarks. Each landmark is given by two coordinates (name, x,y) in pixels.
+    Each keypoint is defined in a similar manner to human labelers marking 2D face kepoints.
     
-    **Type**: `ndarray[float64]`. **Dimensions**: (68, 2).
+    **Type**: `List[Type[str, int, int]`. Should have 68 points.
+    """
+    LANDMARKS_KINECT_V2 = auto()
     """
-    LANDMARKS_3D = auto()
+    Kinect v2 landmarks. Each landmark by name and two coordinates (name, x,y) in pixels.
+
+    **Type**: `List[Type[str, int, int]`.
+    """
+    LANDMARKS_MEDIAPIPE = auto()
+    """
+    MediaPipe Pose landmarks. Each landmark is given by name and two coordinates (name, x,y) in pixels.
+
+    **Type**: `List[Type[str, int, int]`.
+    """
+    LANDMARKS_COCO = auto()
+    """
+    COCO Whole Body landmarks. Each landmark is given by name and two coordinates (name, x,y) in pixels.
+
+    **Type**: `List[Type[str, int, int]`.
+    """
+    LANDMARKS_MPEG4 = auto()
+    """
+    MPEG4 landmarks. Each landmark is given by name and two coordinates (name, x,y) in pixels.
+
+    **Type**: `List[Type[str, int, int]`.
+    """
+    LANDMARKS_3D_IBUG68 = auto()
```

Comment:
Let's update 3D landmarks to tuple format as well.

Reply:
done